README.md

```diff
@@ -6,9 +6,9 @@ _A quintessentially American map style_ [🗺 View the map](https://zelonewolf.g
 
 The purpose of the Americana style is to:
 
-- Express the American experience through cartography
-- Challenge the status quo by showcasing innovation and invention
 - Promote collaboration and common purpose in the American mapping community
+- Express the American experience through cartography, taking inspiration from the familiar features of North American paper maps
```

Comment:
Expressing the American Experience through cartography is a catchy slogan worthy of PBS. ❤️

What would clarify the purpose even better than this section would be a statement affirming the intended audience and geographical scope. https://github.com/ZeLonewolf/openstreetmap-americana/issues/372#issuecomment-1150419745 posed a good question about the degree to which we’re focused on the U.S. versus the rest of the world and how much of the U.S. Additionally, we’re already off the far end of roadgeekery with our shield work, but we also have railfans and trailfans in our midst eager to have a piece of the action. This is FAQ-level material.

Also separately from this mission statement, we should document our sources of inspiration in more detail. Already, we’ve taken inspiration not only from North American road atlases but also from political and topographic maps. Potentially in the near future, trail maps and tourist maps can be added to the mix too. This is important to highlight because our process is fundamentally different than some other OSM projects: many of us don’t put a lot of stock in our own design skills, so we’re actively (re)learning regional cartographic conventions and applying them in every discussion. Familiarity is a shortcut to usable and sensible design.

Reply:
Do you see an additional statement of scope and/or design/inspiration belonging in the lede section or further down?

Reply:
I think we can continue this line of thinking in #385 to clarify what’s being said here. No concerns on my end.